src/style.css

```diff
@@ -0,0 +1,45 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-position: center;
+  background-repeat: no-repeat;
+  background-image: url(images/star.svg);
+  margin-left: 4px;
+  width: 16px;
+  height: 16px;
+}
+
+.stars :nth-child(-n + 1) {
+  margin: 0;
+}
+
+.stars--1 :nth-child(-n + 1) {
```

Comment:
```suggestion
.stars--1 :nth-child(-n + 1),
.stars--2 :nth-child(-n + 2),
...etc {
```
<img width="1113" alt="Screenshot 2023-08-23 at 16 48 43" src="https://github.com/mate-academy/layout_stars/assets/135232001/b56f84ef-9887-4b17-8407-d503c7753010">